docs/live_update_reference.md

```diff
@@ -113,13 +111,39 @@ docker_build('my-img', '.', live_update=[
 2. change to `./src/web/yarn.lock` => run the Live Update; run `setup.sh`; run `cd /app/web && yarn install`, because this file matches that command's trigger
 3. change to `./configs/foo.yaml` => whoops, this file doesn't match any `sync` steps! Even though it matches a trigger (for the third `run`), we won't do a Live Update for this change; instead, we do a full Docker build (see notes on `sync`, above, for what changes trigger a Live Update vs. a full build + deploy).
 
-### [`restart_container()`](api.html#api.restart_container)
+## Rerunning your Process
```

Comment:
I don't know what "rerunning" means here. let's use "restart" to be consistent with how we use it elsewhere.